vignettes/GeomxTools_RNA-NGS_Analysis.Rmd

```diff
@@ -113,6 +118,16 @@ library(GeomxTools)
 library(GeoMxWorkflows)
 ```
 
+```{r checking package versions, eval = FALSE}
```

Comment:
Do we want to restrict users like this? Check with Prajan if OK. Maybe add caveat to use earlier version of bioc released workflow if keeping old version of GxT.

Reply:
@maddygriz any update on this one?

Reply:
@NicoleEO I didn't ask Prajan but I just changed the logic to make sure the WorkFlows and GxT versions are compatible. I'm making sure the vignette knits and then I'll push my changes